### DISCUSSION TITLE
Downloading user comments on other posts?

### DISCUSSION THREAD

**Discussion by mknee354** [MalloyDelacroix/DownloaderForReddit#338] [Q&A]
Hi, I was a bit confused when using this, perhaps I didn't read through the description of the program thoroughly enough, but is it possible to download all the comments of a user that they have made not just on their own posts, but on other people's posts on reddit? And their replies in comment threads? It seemed like this was possible, but I'm trying it, and does not seem to work. If this isn't a feature, it would be a nice one, although completely understand if it's too much to implement.

Thanks!

**Comment by MalloyDelacroix** [comment]
Hello, currently the application only downloads user comments from posts being downloaded or extracted from that user.  Downloading a user's comments and any content linked in those comments is a feature planned for the future.
